### DISCUSSION TITLE
Please tell me your favorite Music Artist.

### DISCUSSION THREAD

**Discussion by s1534** [NAIST-SE/AttractingContributors#441] [General]
Please tell me your favorite Music Artist.
あなたの好きな音楽アーティストを教えてください．

Help me plz!

My fork is here↓
https://github.com/s1534/AttractingContributors-MusicArtist

**Comment by CodyBra** [comment]
I have so many favorite artists that it will be very difficult to remember them all. Okay, let's start with school. When I was 8, I got into a group of emo skaters who listened to emocore and metalcore. That's when I started listening to Bring me the Horizon and Asking Alexandria, and I still love them. Later, when I grew up, I started listening to Nu Metal. From that moment on, I just love Linkin Park, SOAD and Drowning Pool. When I was graduating from school, I realized that I was fed up with everything and started listening to grunge. Absolutely all the grunge that I found. Right now I like to listen to Funeral Dinner, Oceans Ate Alaska, the Ganjas, Sum 41, King Gizzard and the Lizzard Wizard and so on!

**Comment by kaircx** [comment]
help me..
https://github.com/kaircx/AttractingContributors-favorite-language/
